spring-boot/src/test/java/org/dcsa/conformance/springboot/ConformanceBasicAPITest.java

```diff
@@ -88,28 +82,6 @@ void shouldReturnPrintableReport() throws Exception {
       .andExpect(content().string(containsString("SupplyCSP [REGULAR]")));
   }
 
-  @Test
```

Comment:
Already removed these, since the entry points are going to removed as well, as we discussed.